_stbt/core.py

```diff
@@ -1622,10 +1637,14 @@ def teardown(self):
             source = None
         if not self.novideo:
```

Comment:
Why `NoVideo` which is caused by the source pipeline should have an effect on the `sink_pipeline` I don't know